pom.xml

```diff
@@ -103,7 +103,12 @@
         <dependency>
             <groupId>io.springfox</groupId>
             <artifactId>springfox-swagger2</artifactId>
-            <version>2.6.1</version>
+            <version>2.9.2</version>
+        </dependency>
+        <dependency>
```

Comment:
I don't see were this dependency is imported; is it really used?

Reply:
@npasquetto do we?

Reply:
Just to be clear I was asking about the google.guava dependency at lines 109-110

Reply:
good point, I understood springfox-swagger2 indeed 😆

Reply:
No: I've introduced google.guava dependency as a workaround for a bug in a previous version of springfox2 (https://github.com/springfox/springfox/issues/2232) that I've tested before the MR submission, but in the springfox2 version used in the MR, the bug it's been fixed, so the workaround (and the dependency) it's no longer needed.